Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:10
```

Comment:
This shouldn't be at the top level of the repo, it belongs inside its package.  Failing that, it will need a very specific filename.

Reply:
Change to `testing.Dockerfile`.